logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/client/UpdateSelfClientCapabilityToConsumableNotificationsUseCase.kt

```diff
@@ -52,10 +52,9 @@ internal class UpdateSelfClientCapabilityToConsumableNotificationsUseCaseImpl in
     private val selfServerConfig: SelfServerConfigUseCase,
     private val syncRequester: suspend () -> Either<CoreFailure, Unit>,
     private val slowSyncRepository: SlowSyncRepository,
+    private val logger: KaliumLogger
```

Comment:
It would be nice to also inject this in EventRepository. Just to be sure we can split logs from multiple accounts in a single device.